testsuite/MDAnalysisTests/coordinates/test_writer_api.py

```diff
@@ -83,14 +82,17 @@ def test_write_with_atomgroup(writer, tmpdir):
         pytest.skip("MOL2 only writes MOL2 back out")
     elif writer == mda.coordinates.LAMMPS.DATAWriter:
         pytest.skip("DATAWriter requires integer atom types")
-    elif writer == mda.coordinates.H5MD.H5MDWriter:
-        if not mda.coordinates.H5MD.HAS_H5PY:
-            pytest.skip("macOS and ubuntu, 3.6, MIN tests fail")
+    elif writer == mda.coordinates.H5MD.H5MDWriter and not mda.coordinates.H5MD.HAS_H5PY:
+        pytest.skip("macOS and ubuntu, 3.6, MIN tests fail")
```

Comment:
It's not a fail, just a skip. Just say "skipping for MIN dependency test"

Reply:
Also, remove 3.6, this might change in the future. ... and the ubuntu/macOS
```suggestion
        pytest.skip("skipping H5MDWriter test because h5py is not installed")
```